.changeset/fresh-bats-laugh.md

```diff
@@ -0,0 +1,5 @@
+---
+"gatsby-source-strapi": major
+---
+
+add gatsby-source-strapi
```

Comment:
Let's just add a little more detail here on the change in developers and added Gatsby 5 support.

Reply:
Ah, as this will end up in the releases comments? I've never used this kind of changeset before. Also, I'm not 100% sure it will work in Gatsby 5, I haven't upgraded myself yet haha. But someone in the comments said it did so assuming my way to victory here 😇 . Can we do a pre-release or something?

Reply:
Yeah, I was assuming that comment. We can, I haven't tried it and it's not straight forward. I'd just make a note that the lack of tests mean there could be an issue

Reply:
It's a major so folks should be cautious.

Reply:
Basically I'm okay yoloing this. If you want to be more cautious then go for it.

Reply:
Yes, that note ends up in the release.

Reply:
Yeah specially for the gasby 5 thing right. If it doesn't work correctly on 5, i won't in the current as well. Only thing we did is set peer deps.